### DISCUSSION TITLE
v2 2주 완성 계획

### DISCUSSION THREAD

**Discussion by scarf005** [jiphyeonjeon-42/backend#625] [Announcements]
https://chrome.google.com/webstore/detail/notifier-for-github/lmjdlojahmbbcodnpecnjnmlddbkjhnn
https://addons.mozilla.org/en-US/firefox/addon/notifier-for-github/

**Comment by jimin52** [comment]
구독 좋아요 알림설정 했습니다.

**Comment by Yonaim** [comment]
와~
